src/visualizer/codeinformation/codeexplaination/CodeExplanation.js

```diff
@@ -52,7 +52,9 @@ const CodeExplanation = ({ algo }) => {
 
     return (
       <div className="performance">
-        <p className="explanationPara">Assuming size of array is N:</p>
+        <p className="explanationPara">
```

Comment:
whats the formatting we're following for css classNames? "-" between words or camel case will do as well?